src/app/editor/components/node-editor/node-editor.component.ts

```diff
@@ -141,6 +141,10 @@ export class NodeEditorComponent implements OnInit, OnDestroy {
         return !!this.node && !/\.0$/.test(this.node.version);
     }
 
+    isSaving(): boolean {
```

Comment:
Might it be better to expose an observable, `saving$`, rather than calling a method with each change detection cycle?